app/browser/reducers/historyReducer.js

```diff
@@ -60,19 +55,15 @@ const historyReducer = (state, action, immutableAction) => {
         if (Immutable.List.isList(action.get('historyKey'))) {
           action.get('historyKey', Immutable.List()).forEach((key) => {
             state = historyState.removeSite(state, key)
+            // TODO: Remove history site Sync
```

Comment:
ah, it would be more clear if it were 'Sync history site removal', in this case i thought you were removing history site sync